cmd/integration/integration.go

```diff
@@ -295,6 +302,9 @@ func podNotFound(c *client.Client, podNamespace string, podID string) wait.Condi
 func podRunning(c *client.Client, podNamespace string, podID string) wait.ConditionFunc {
 	return func() (bool, error) {
 		pod, err := c.Pods(podNamespace).Get(podID)
+		if apierrors.IsNotFound(err) {
```

Comment:
If error terminates the poll, shouldn't we always return no err? What if the apiserver refuses the connection or some other unrelated errors happen?